src/interactableComponents/drawer/index.js

```diff
@@ -462,6 +467,12 @@ export default class Drawer extends BaseComponent {
     );
   }
   render() {
+    const {migrate} = this.props;
+
+    if (migrate) {
```

Comment:
Why use 'migrate' prop and not replace the component?

Reply:
to allow us control the migration. 
Also this way we can migrate only in private uilib. 
for the public it will be better to migrate when we upgrade major version